build.assets/Dockerfile

```diff
@@ -268,11 +274,13 @@ RUN helm plugin install https://github.com/vbehar/helm3-unittest && \
     HELM_PLUGINS=/home/ci/.local/share/helm/plugins helm plugin list
 
 # Install JS gRPC tools.
+# Keep the version below synced with devbox.json
```

Comment:
Those versions aren't declared/updated here, move the comment to the Makefile?

Same for other args.